clang/include/clang/CIR/Dialect/IR/CIRTypes.td

```diff
@@ -193,7 +193,7 @@ def CIR_LongDouble : CIR_FloatType<"LongDouble", "long_double"> {
 
 // Constraints
 
-def CIR_AnyFloat: AnyTypeOf<[CIR_Single, CIR_Double, CIR_FP80, CIR_LongDouble]>;
+def CIR_AnyFloat: AnyTypeOf<[CIR_Single, CIR_Double, CIR_FP80, CIR_LongDouble, CIR_FP16, CIR_BFloat16]>;
```

Comment:
nit: format to make it at most 80 columns wide

Reply:
Done. The .td file is not formated automatically.